plugins/email_service/app/controllers/email_service/__configsets_controller.rb

```diff
@@ -0,0 +1,113 @@
+# module EmailService
```

Comment:
Do you really need to keep a copy of a controller? You can allways check the git history